index.html

```diff
@@ -14,166 +14,171 @@
-                  can get to work, manage child care and do their shopping.
-                </p>
-                <p class="article__read-more">
-                  <a href="#">
-                    <span class="arrow-right"></span>
-                    Learn more
-                  </a>
-                </p>
-              </div>
-
-              <div class="article">
-                <h3 class="article__title">How can I help?</h3>
-                <p class="article__summary">
-                  We need lots of bikes and bike accessories! If you have an
-                  old bike you don't use, bring it to one of our dropoff
-                  events. Or come attend one of our fundraisers.
-                </p>
-                <p class="article__read-more">
-                  <a href="#">
-                    <span class="arrow-right"></span>
-                    Learn more
-                  </a>
-                </p>
-              </div>
-            </div>
-          </div>
-        </div>
-
-        <div class="sidebar">
-          <h2 class="heading-underline">Upcoming events</h2>
-
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="spring-fundraisers_thumbnail.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Join us for our Spring fundraisers</a>
-              </h3>
-              <p class="article__summary">
-                Riders needed now to help raise awareness and funds for BfR and
-                Médecins Sans Frontières.
-              </p>
-            </div>
-          </div>
-
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="bikes-for-refugees_logo.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Bike dropoff event in Edinburgh on May 11</a>
-              </h3>
-            </div>
-          </div>
-
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="edinburgh-damascus_thumbnail.png"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Help us cycle 4,797km</a>
-              </h3>
-              <p class="article__summary">
-                We need you to help us cycle 4797km, the distance of Edinburgh
-                to Damascus.
-              </p>
-            </div>
-          </div>
+          </section>
+        </section>
+
+        <a
+          href="https://www.facebook.com/BikesforRefugeesScotland/events/?ref=page_internal"
+          target="_blank"
+          class="facebook-link"
```

Comment:
This link is on the right on your page, but it is on the left in the picture! try adding display:flex
and then play around with justify-content:  to put it on the left